dartagnan/src/main/java/com/dat3m/dartagnan/expression/integers/IntLiteral.java

```diff
@@ -28,7 +28,7 @@ public BigInteger getValue() {
     }
 
     public int getValueAsInt() {
-        return value.intValueExact();
+        return value.intValue();
     }
```

Comment:
Why this change? We want to throw an exception here cause that means we should not have used the method in the first place. With the change, truncation errors are swallowed silently. Did you encounter an issue with this?